### PR TITLE
write statistics page

### PR DESCRIPTION
**Pull request by Eric-ZheranLi**
Write statistics page

image

image

### CODE REVIEW

File: src/components/StatisticsContent/index.tsx [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+    sprint_best_series: 7,
+    'audio-call_words_studied': 5,
+    'audio-call_correct_answers': 90,
+    'audio-call_best_series': 7,
+  };
+
+  const page_data = pageType === 'For the day' ? today_statistics : all_time_statistics;
+
+  return (
+    <main className="flex flex-col items-center justify-center gap-[40px] bg-tk_greyLight py-[100px]">
```
Comment (Double-Twelve) [inline src/components/StatisticsContent/index.tsx:34]: don't need `main` tag

File: src/components/StatisticsContent/index.tsx [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+              ? 'underline underline-offset-[16px]'
+              : 'text-tk_greyDark'
+          }
+          onMouseEnter={() => setPageType('For all the time')}
+        >
+          For all the time
+        </h4>
+      </section>
+      <section className="flex gap-[10px]">
+        <div className="flex h-[511px] w-[583px] flex-col gap-[20px]  rounded-[14px] bg-white p-[40px]">
```
Comment (Double-Twelve) [inline src/components/StatisticsContent/index.tsx:56]: don't need to set width and height

File: next.config.js [hunk 3]
```diff
@@ -31,6 +31,10 @@ const nextConfig = {
 
     return config;
   },
+
+  images: {
+    unoptimized: true,
+  },
```
Comment (Double-Twelve) [inline next.config.js:37]: why do you need to add this?

**Review by jasonzhwang — APPROVED** [review]
LGTM

**Pull request merged**